plasmapy/formulary/collisions.py

```diff
@@ -2100,29 +2141,24 @@ def __init__(
         field_particle : |Particle|
             The background particle being interacted with.
 
-        v_a : `~astropy.units.Quantity`, optional
-            The relative speed between particles. If not provided, T_a must be specified and
-            thermal velocity is assumed: :math:`μ v_a^2 \sim 2 k_B T_a` where
+        v_drift : `~astropy.units.Quantity`, optional
+            The relative drift between the test and field particles. If not provided, T_a must be specified and
+            thermal velocity is assumed to be dominant: :math:`μ v_a^2 \sim 2 k_B T_a` where
             :math:`μ` is the reduced mass. Cannot be negative.
 
-        T_a : `~astropy.units.Quantity`, optional
-            The temperature of the test species. Only necessary if :math:`v_a` is not provided.
-
-        n_b : `~astropy.units.Quantity`
-            The number density of the background field particles in units convertible to :math:`\frac{1}{m^{3}}`
-
-        T_b : `~astropy.units.Quantity`
+        T_b : `~astropy.units.Quantity`, optional
             The temperature of the background field particles in units convertible to degrees Kelvin.
 
+        n_b : `~astropy.units.Quantity`, optional
+            The number density of the background field particles in units convertible to :math:`\frac{1}{m^{3}}`.
+
         Coulomb_log : `~astropy.units.Quantity`
             The value of the Coulomb logarithm evaluated for the two interacting particles.
```

Comment:
"two interacting particles" seems wrong since it's a test particle interacting with a _population_ of field particles (more than 1)

```suggestion
            The value of the Coulomb logarithm for the interaction.
```